packages/amazonq/package.json

```diff
@@ -2,7 +2,7 @@
     "name": "amazon-q-vscode",
     "displayName": "Amazon Q",
     "description": "The most capable generative AI-powered assistant for building, operating, and transforming software, with advanced capabilities for managing data and AI",
-    "version": "1.83.0-SNAPSHOT",
```

Comment:
I believe this must stay as SNAPSHOT. Is there we reason we want to change it here?